src/components/Icon/index.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import styled from 'styled-components';
+
+import * as SVGS from '../../constants/icons';
+
+const NAMES = Object.keys(SVGS);
+
+const ICONS = NAMES.reduce(
+  (icons, icon) => ({
```

Comment:
Nice for when we add new icons :heavy_plus_sign:

Reply:
this was there before, kudos to Ugo